Mathlib/Analysis/NormedSpace/LpEquiv.lean

```diff
@@ -81,6 +81,7 @@ theorem equiv_lpPiLp_norm (f : lp E p) : ‖Equiv.lpPiLp f‖ = ‖f‖ := by
 
 /-- The canonical `AddEquiv` between `lp E p` and `PiLp p E` when `E : α → Type u` with
 `[Fintype α]` and `[Fact (1 ≤ p)]`. -/
+@[nolint unusedArguments]
 def AddEquiv.lpPiLp [Fact (1 ≤ p)] : lp E p ≃+ PiLp p E :=
```

Comment:
I can't explain why this didn't trip the linter before, but this argument really isn't necessary.
```suggestion
`[Fintype α]`. -/
def AddEquiv.lpPiLp : lp E p ≃+ PiLp p E :=
```

Reply:
I just pushed this change myself.